executor/common_linux.h

```diff
@@ -1094,8 +1094,10 @@ static int hwsim80211_create_device(int sock, int hwsim_family, uint8 mac_addr[E
 	netlink_attr(&msg, HWSIM_ATTR_SUPPORT_P2P_DEVICE, NULL, 0);
 	netlink_attr(&msg, HWSIM_ATTR_PERM_ADDR, mac_addr, ETH_ALEN);
 	int err = netlink_send(&msg, sock);
-	// TODO: in case of success hwsim returns index instead of 0. Fix it.
-	if (err < 0) {
+	// TODO: in case of success, hwsim returns index instead of 0, while netlink_send_ext supposes
```

Comment:
(I think this commit should be folded into the original commit that introduced that change. If things are still WIP, please ignore this comment.)

Reply:
Yes, at the moment thigns are temporarily WIP. I'll squash commits after I complete work